File: 02-09-26/3.py
import numpy as np

n = int(input("Enter the value of n: "))
values = np.array(input("Enter the values: ").split(), dtype=int)

# Count occurrences of each integer from 0 to 9
counts = np.bincount(values, minlength=10)

print()
for digit in range(10):
    print(f"{digit}: {counts[digit]}")

File: 02-09-26/3.c
#include<stdio.h>
int main(){
	int n;
	printf("enter a positive integer n: \n");
	scanf("%d",&n);
	int values[n];
	for(int i=0;i<n;i++){
	printf("enter the values in between 0 & 9: ");
	scanf("%d",&values[i]);
	}
	int count=0,i=0;
        if(values[i]==0){
		count = count +1;
		i++;
	return 0;
}
